### PR TITLE
Fastapi default api and Llama3.1 405B update

### PR DESCRIPTION
**Pull request by snova-luiss**
- Update config files / streamlit apps and make Fast API the default
- Update notebooks and make Fast API the default

### CODE REVIEW

**Comment by snova-petrojm** [comment]
Thanks, @snova-luiss. We're missing the changes for the following kits: prompt_engineering and post_call_analysis

**Comment by snova-luiss** [comment]
Remaining notebooks:
- Benchmarking, Edgar
- Prompt engineering (waiting on refactor)
- Function calling (fixing the issues, one left query_db)
- PCA (blocked)

File: enterprise_knowledge_retriever/config.yaml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 api: "fastapi" #  set either sambastudio, sambaverse or fastapi
 
 embedding_model: 
-    "type": "cpu" # set either sambastudio or cpu
+    "type": "sambastudio" # set either sambastudio or cpu
```
Comment (snova-petrojm) [inline enterprise_knowledge_retriever/config.yaml:4]: we want to keep cpu as default here

File: multimodal_knowledge_retriever/config.yaml [hunk 2]
```diff
@@ -8,11 +8,11 @@ embedding_model:
...
 llm: 
     "temperature": 0.0
+    "do_sample": False
     "max_tokens_to_generate": 1200
-    "do_sample": false
-    "sambaverse_model_name": "Meta/llama-2-70b-chat-hf"
+    "sambaverse_model_name": "Meta/Meta-Llama-3-8B-Instruct"
     "coe": True #set as true if using Sambastudio CoE endpoint
-    "select_expert": "llama-2-70b-chat-hf" #set if using Sambaverse or SambaStudio CoE llm expert
+    "select_expert": "llama3-405b" #set if using FastCoE, Sambaverse or SambaStudio CoE llm expert
```
Comment (snova-petrojm) [inline multimodal_knowledge_retriever/config.yaml:15]: should be FastAPI instead of FastCoE (but will be removed anyways)

**Review by snova-petrojm — APPROVED** [review]

**Review by snova-rodrigom — APPROVED** [review]
lgtm. I know there are some remaining notebooks pending for this change, so that's ok

**Pull request merged**